src/NodeRunner.ts

```
@@ -176,6 +181,39 @@ export default class NodeRunner {
     }
   }
 
+  private async savePricePackage(signedPrices: PriceDataSigned[]) {
+    logger.info("Saving price package");
```

What do you think about wrapping only signing in try-catch. We could then 
1) have separate time tracking for signing
2) catch separately errors for signing (broadcasting has got an own try-catch block)

---

I would not wrap signing in try-catch block, because if package signing doesn't work we want to throw an exception and stop the package broadcasting process. If we add signing in try-catch we will need to handle it manually and the code would be less readable IMO. We still will be able to analyze package signing timings, because we have a timing for the whole process and a separate timing for broadcasting